easybuild/easyconfigs/n/numactl/numactl-2.0.11-iccifort-2016.3.210-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'numactl'
+version = '2.0.11'
+
+checksums = ['d3bc88b7ddb9f06d60898f4816ae9127']
+
+homepage = 'http://oss.sgi.com/projects/libnuma/'
+description = """The numactl program allows you to run your application program on specific cpu's and memory nodes.
+ It does this by supplying a NUMA memory policy to the operating system before running your program.
+ The libnuma library provides convenient ways for you to add NUMA memory policies into your own program."""
+
+toolchain = {'name': 'iccifort', 'version': '2016.3.210-GCC-5.4.0-2.26'}
+
+source_urls = ['ftp://oss.sgi.com/www/projects/libnuma/download/']
+sources = [SOURCE_TAR_GZ]
+
+# builddependencies = [('Autotools', '20150215', '', ('GCC', '5.4.0-2.26'))]
+
+# preconfigopts = './autogen.sh && '
```

Comment:
Honest answer: I have forgotten, did that a few month ago when I had used .  So far it seems to work.  

If we want/need to include those, which ones would you recommend? Nothing in EB 2.8.2 seems matching (there are foss and intel).  I assume we would need to base one on GCCcore 5.4.0 for the toolchain in question.  So would that be ('GCC', '5.4.0-2.26') or ('GCCcore', '5.4.0-2.26')?  Is that somewhere in the pipeline in the developments for EB 2.9 such that we could lift it here?